src/editor/gltf/GLTFLoader.js

```diff
@@ -192,6 +192,11 @@ const GLB_HEADER_MAGIC = "glTF";
 const GLB_HEADER_LENGTH = 12;
 const GLB_CHUNK_TYPES = { JSON: 0x4e4f534a, BIN: 0x004e4942 };
 
+const HUBS_NODEREF_COMPONENTS = {
```

Comment:
I'd probably move this map somewhere else and import it into this GLTFLoader file. I think it'll be easier to maintain that way. Ideally it'll eventually be a glTF plugin, so maybe put it where you'll eventually write that in a separate file in this glTF folder.